<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactSupportDialog</class>
 <widget class="QDialog" name="ContactSupportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>801</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="ContactSupportDialogGroupBox">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Contact Support Form</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="ContactSupportDialogDateTimeStampLabel">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Date/Time Stamp</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="ContactSupportDialogCompanyNameLabel">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Company Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="ContactSupportDialogVerticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="ContactSupportDialogCompanyAddressLine1Label">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Company Address Line 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="ContactSupportDialogUserNameLabel">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>User's Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="ContactSupportDialogCompanyAddressLine2Label">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Company Address Line 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="ContactSupportDialogUserEmailLabel">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>User's Email</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="ContactSupportDialogCompanyPhoneNumberLabel">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Company Phone Number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="ContactSupportDialogUserTypeLabel">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>User Type</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="Line" name="ContactSupportDialogSeperatorLine">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="ContactSupportDialogCategoryLabel">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Category</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="ContactSupportDialogCategoryComboBox">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Select A Category</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data Logger (Damaged)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data Logger (WiFi Connectivity)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data Logger (Database Connectivity)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data Logger (Power)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Database (Connectivity)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Database (Server Crash)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Application (Login/Logout)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Application (User Controls)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Application (Device Management)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Application (Reports)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Application (Settings) </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Additional Feedback</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Suggestions</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="ContactSupportDialogDetailedDescriptionLabel">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Please provide a detailed description</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="ContactSupportDialogCharacterMinimumLabel">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>250 character minimum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QDialogButtonBox" name="ContactSupportDialogButtonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <widget class="QTextEdit" name="ContactSupportDialogTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
